jscomp/syntax/bs_syntaxerr.ml

```diff
@@ -57,11 +57,11 @@ type error
 let pp_error fmt err =
   Format.pp_print_string fmt (match err with
   | Bs_uncurried_arity_too_large
-    -> "BuckleScript support uncurried function up to arity 22"
+    -> "Uncurried function supports only up to arity 22"
   | Label_in_uncurried_bs_attribute
-    -> "BuckleScript uncurried function doesn't support labeled arguments yet"
+    -> "Uncurried function doesn't support labeled arguments yet"
```

Comment:
@bobzhang uncurried functions do support labeled arguments now; how come this is still used?

Reply:
indeed, I will double check